mundane-bytes/src/main/scala/com/ambiata/mundane/bytes/Buffer.scala

```diff
@@ -67,6 +65,9 @@ object Buffer {
     wrapArray(b.bytes, b.offset + o, b.length - o)
   }
 
+  def reset(b: Buffer): Unit =
```

Comment:
> , I don't actually care that much - and I kind of see why it is different, just thought I would ask to see if you had thought about it that much.

Yeah originally I had it return `Buffer`, but then it seemed like this type was more precise. Happy to change it if you prefer.